Formula/v/vulkancapsviewer.rb

```diff
@@ -0,0 +1,30 @@
+class Vulkancapsviewer < Formula
+  desc "Client application to display hardware implementation details for Vulkan GPUs"
+  homepage "https://github.com/SaschaWillems/VulkanCapsViewer"
+  url "https://github.com/SaschaWillems/VulkanCapsViewer.git", tag: "3.32", revision: "3024e3148a7b47706b6d259f8e0ad0a2aa651330"
```

Comment:
Do we need a Git clone? Can we use a tarball?

Reply:
It would be possible to use a tarball, but would there be a specific reason for it?
The Formula CookBook doesn't seem to recommend either way when both are available, and the git clone seems to me to be simpler to specify and maintain.

Is this a performance consideration? I don't have my own numbers on this, nor a very recent analysis, but according to this post and following discussion: https://blog.verbum.org/2012/06/08/efficiency-of-git-versus-tarballs-for-source-code-transmission-and-storage-over-time/
* if we cache the git repo so that we are not performing a whole clone every time the package is built, this can result in significant savings due to only downloading the delta.
* If we don't, we can use a shallow clone and get similar overhead to a tarball.

Reply:
> if we cache the git repo so that we are not performing a whole clone every time the package is built, this can result in significant savings due to only downloading the delta.

We don't since it runs on multiple machines, but we can't use shallow clones either since the performance of those is worse. The best performance is downloading a tarbal.